test/consts/helm.go

```diff
@@ -0,0 +1,4 @@
+package consts
+
+// KongHelmChartVersion is the version of the Kong Helm chart to use in tests.
+const KongHelmChartVersion = "2.26.0"
```

Comment:
I think that falls into a wider range of dependencies we have in tests and we have to update them manually. There's https://github.com/Kong/kubernetes-ingress-controller/issues/1886 that we could make handle this const's update as well.

Reply:
"nice-to-have" and "priority/low" are definitely labels that will make this issue stand out and be prioritized :)

Thanks for digging it out though!

Reply:
Let's bring it to the next sync and prioritize it maybe. It's the pain we all have to struggle with when releasing KIC. It would also make compatibility bugs to be discovered quicker.